CS/ContextMenu_CS.txt

```diff
@@ -1,466 +1,481 @@
+    ContextMenu_PatchHole = "Zašít díru",
+    ContextMenu_PickupBrokenGlass = "Sebrat rozbité sklo",
+    ContextMenu_Pink = "Růžová",
+    ContextMenu_PlaceAll = "Položit vše",
+    ContextMenu_PlaceHalf = "Položit polovinu",
+    ContextMenu_PlaceItemOnGround = "Položit předmět na zem",
+    ContextMenu_PlaceOne = "Položit jeden",
+    ContextMenu_Place_Fishing_Net = "Položit rybářskou síť",
+    ContextMenu_Place_Trap = "Nastražit past",
+    ContextMenu_PlantainCataplasm = "Jitrocelový obklad",
+    ContextMenu_Plaster = "Omítka",
+    ContextMenu_PlumbItem = "Srovnat %1",
+    ContextMenu_Pour_into = "Nalít do",
+    ContextMenu_Pour_on_Ground = "Vylít na zem",
+    ContextMenu_Purple = "Fialová",
+    ContextMenu_PutCombinationPadlock = "Použít číselný visací zámek",
+    ContextMenu_PutInContainer = "Vložit do úložiště (%1)",
+    ContextMenu_PutOnEarlobe = "Nasadit na ucho",
+    ContextMenu_PutOnEartop = "Nasadit na vršek ucha",
+    ContextMenu_PutPadlock = "Použít visací zámek",
+    ContextMenu_Put_in_Container = "Vložit do úložiště",
+    ContextMenu_Put_out_fire = "Uhasit oheň",
+    ContextMenu_Put_up_tent = "Postavit stan",
     ContextMenu_Rack = "Natáhnout %1",
-    ContextMenu_UnloadRoundFrom = "Vybít 1 střelu z %1",
+    ContextMenu_Rain_Collector_Barrel = "Nádoba na vodu",
+    ContextMenu_Read = "Přečíst",
+    ContextMenu_Read_Note = "Přečíst %1",
+    ContextMenu_Red = "Červená",
+    ContextMenu_Reload = "Přebít",
+    ContextMenu_Remove = "Odstranit",
+    ContextMenu_RemoveAllPatches = "Odstranit všechny záplaty",
+    ContextMenu_RemoveBrokenGlass = "Odstranit rozbité sklo",
+    ContextMenu_RemoveBurntVehicle = "Rozebrat vrak auta",
+    ContextMenu_RemoveBush = "Odstranit keř",
+    ContextMenu_RemoveCharcoal = "Odebrat dřevěné uhlí",
+    ContextMenu_RemoveCombinationPadlock = "Odstranit číselný visací zámek",
+    ContextMenu_RemoveFromHotbar = "Odebrat %1 z %2",
+    ContextMenu_RemoveGrass = "Vytrhat trávu",
+    ContextMenu_RemoveLightbulb = "Odstranit žárovku",
+    ContextMenu_RemovePadlock = "Odstranit visací zámek",
+    ContextMenu_RemovePatch = "Odstranit záplatu",
+    ContextMenu_RemovePlayerFailed = "Nemůžeš odebrat hráče, který je blízko základně.",
+    ContextMenu_RemoveWallVine = "Odstranit břečťan",
+    ContextMenu_Remove_Bait = "Odebrat návnadu",
+    ContextMenu_Remove_Bandage = "Odstranit obvaz",
+    ContextMenu_Remove_Battery = "Odstranit baterii",
+    ContextMenu_Remove_Bullet = "Odstranit náboj",
+    ContextMenu_Remove_Glass = "Odstranit sklo",
+    ContextMenu_Remove_Propane_Tank = "Odebrat propanovou lahev",
+    ContextMenu_Remove_Splint = "Odstranit dlahu",
+    ContextMenu_Remove_Stitch = "Odstranit steh",
+    ContextMenu_Remove_Trap = "Odstranit past",
+    ContextMenu_Remove_Weapon_Upgrade = "Odebrat vylepšení",
+    ContextMenu_Remove_curtains = "Odstranit záclony",
+    ContextMenu_Remove_escape_rope = "Odstranit únikové lano",
```

Comment:
Tohle mi příjde tak blbý. nestačilo by "Odstranit lano"?
To samé níže
"Odstranit látkové lano"